NBXplorer.Client/ExplorerClient.cs

```diff
@@ -313,7 +313,7 @@ public void CancelReservation(DerivationStrategyBase strategy, KeyPath[] keyPath
 			CancelReservationAsync(strategy, keyPaths, cancellation).GetAwaiter().GetResult();
 		}
 
-		public GetBalanceResponse GetBalance(DerivationStrategyBase userDerivationScheme, CancellationToken cancellation = default)
```

Comment:
the cancellation token should be passed to `GetBalanceAsync`, not removed.

Reply:
Ok, But GetBalance is sync version and cancelationToken don't used inside GetBalanceAsync  call. I'll change it to use

Reply:
this is a bug, we should use it in `GetBalanceAsync `

Reply:
fix was pushed